.github/requirements.txt

```diff
@@ -2,4 +2,13 @@ pylint
 pydocstyle
 flake8
 pytest
-pyinputplus
\ No newline at end of file
+pyinputplus
+pathlib
+askdirectory
+os
```

Comment:
There is no need for putting built-in modules into this file. Its purpose is to list 3rd party libraries, so other people can install the same packages that you have. Please remove from here:

- pathlib
- os
- csv
- shutil

Reply:
built-in modules removed